e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -176,25 +180,54 @@ export class UIhelper {
   }
 
   async verifyColumnHeading(
-    rowTexts: string[] | RegExp[],
+    rowTexts: (string | RegExp | (string | RegExp)[])[],
     exact: boolean = true,
   ) {
     for (const rowText of rowTexts) {
-      const rowLocator = this.page
-        .locator(`tr>th`)
-        .getByText(rowText, { exact: exact })
-        .first();
-      await rowLocator.waitFor({ state: 'visible' });
-      await rowLocator.scrollIntoViewIfNeeded();
-      await expect(rowLocator).toBeVisible();
+      if (Array.isArray(rowText)) {
+        let found = false;
+
+        for (const text of rowText) {
```

Comment:
Nested loop is not needed, you can pass OR operator in regex to verify either of it for eg : /Permission Policies|Accessible plugins/